### DISCUSSION TITLE
Numpy

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#24] [Q&A] [answered]
Can np. array() accept a tuple of values..?

**Comment by sainathch45** [comment, marked as answer]
Yes, np.array() can accept a tuple of values although array or list is generally within square brackets. When we pass a tuple of values using parenthesis in np.array(), it automatically converts the tuple into a list and returns the list only when we try to print the values.

**Comment by 17-Manish** [comment]
yes , if you give np.array() a tuple values around the bracket for eg (1,2,3),it will automatically turn it into a list  it Shows in tuple I.e,[1,2,3] while printing it

**Comment by TRahulsingh** [comment]
Yes, np.array() can accept a tuple of values, converting it into a NumPy array.
